src/language-js/parse/postprocess/index.js

```diff
@@ -144,6 +144,19 @@ function postprocess(ast, options) {
           return node.types[0];
         }
         break;
+
+      // OXC bug?
```

Comment:
Feel free to contact us in https://discord.gg/9uXCAwqQZW or create an issue directly.

Reply:
Thanks, this was added months ago to bypass tests, seems already fixed.